scripts/infra

```diff
@@ -41,6 +41,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
 
                 # Build Lambda function archives
                 make -sC lambda-functions/alert_batch_failures
+                make -sC lambda-functions/alert_sfn_failures
```

Comment:
I see a world where we combine or further abstract these functions, [like we do in Raster Foundry](https://github.com/azavea/raster-foundry-deployment/tree/develop/core/terraform/lambda-functions/alert_failures). I decided not to pursue this since there was already a challenge in parsing the ECS output that's specific to Step Functions. 

Imagine if we had a Python module that's job was just to handle AWS errors and surface them to Slack. I think that could be useful.

Reply:
Agreed, that sounds super useful. I'm kinda surprised that AWS doesn't have some SNS-adjacent feature or product to clean up and send valid notifications out to common chat platforms already. Looks like Lambda is still the way to go!

Reply:
Also, lemme just say, I love Make.